src/App.js

```diff
@@ -1,25 +1,75 @@
 import React from 'react';
 
+import names from './data/names';
+import { Clock } from './components/Clock';
+
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: 0,
+  };
+
+  componentDidUpdate(prevProp, prevState) {
+    if (prevState.name !== this.state.name) {
+      // eslint-disable-next-line no-console
+      console.log(`
+        The Clock was renamed from ${prevState.name} to ${this.state.name}`);
+    }
+  }
+
+  render() {
+    const { isClockVisible, name } = this.state;
+
+    return (
+      <>
+        <div className="current-time">
+          {`Current time: `}
+          {isClockVisible ? <Clock /> : 'hide clock'}
```

Comment:
```suggestion
          {isClockVisible ? <Clock /> : 'hidden'}
```
It's not clear what 'hide clock' should mean. If you want to say that clock is hidden, it's better to write it like this